griptape/drivers/prompt/amazon_bedrock_prompt_driver.py

```diff
@@ -41,6 +43,8 @@
     from griptape.common import PromptStack
     from griptape.tools import BaseTool
 
+logger = logging.getLogger(Defaults.logging_config.logger_name)
+
```

Comment:
instantiating the loggers here blows up the lazy logging config, as these get set on import

Reply:
I think that's fine, the lazy loading was mostly to get around circular dependencies. Should I remove `lazy_property` on `logging_config`?

Reply:
well i think the loggers need to be instantiated later. as implemented, if you update any properties on the logging config via `Defaults`, nothing will actually get updated because the config gets set on import

Reply:
unless you do all the logging config stuff before importing anything, which seems ugly

Reply:
Or you create a new `LoggingConfig` as I do in the example. Not _great_ but I think it's pretty common practice to have loggers declared up top.

Reply:
does that example work? the logger has already been fetched on import, updating the logging config afterwards wouldnt do anything i think

Reply:
It does work, but I'm realizing it's only because creating a new `LoggingConfig()` calls the `post_init` logic.

I guess only real solution here is to move all instances of logger into the methods.

Reply:
I'm now remembering that we intentionally kept functionality like this out of `LoggingConfig` to avoid unnecessary abstraction.

What about reverting `LoggingConfig` and changing example to:

```python
import logging

from griptape.configs import Defaults
from griptape.configs.logging import JsonFormatter
from griptape.structures import Agent

logger = logging.getLogger(Defaults.logging_config.logger_name)
logger.setLevel(logging.DEBUG)
logger.handlers[0].setFormatter(JsonFormatter())

agent = Agent()

agent.run("Hello world!")
```

Reply:
i kinda like the idea of exposing a `logger` parameter on the config and calling `config.logger` every time, but i know thats not very pythonic

Reply:
That feels out of place for the `Logging*Config*` though. Tbh this class probably should not exist and logger name should just be a constant somewhere else.

Reply:
```
from griptape.drivers import OpenAiChatPromptDriver
```
maybe im misunderstanding how fetching loggers works, but I thought the logger gets fetched on import and any subsequent changes to the logger would not get picked up.

Reply:
> That feels out of place for the `Logging*Config*` though. Tbh this class probably should not exist and logger name should just be a constant somewhere else.

```
from griptape.config import LOGGER_NAME
```

Reply:
> maybe im misunderstanding how fetching loggers works, but I thought the logger gets fetched on import and any subsequent changes to the logger would not get picked up.

`logger = logging.getLogger(...)` always refers to the same griptape logger. So changing values like debug level, formatter, etc post init is fine.